README.md

```diff
@@ -1 +1,49 @@
-# node-hello-world-lab
\ No newline at end of file
+# Node Hello World
+
+## Objectives
+
+1. Implement Hello World in Node
+1. Run the script from the terminal
+
+
+## Introduction
+
+In this lab, you'll practice how to run Node scripts from command line. In addition, you'll develop your first Node application—Hello World, the traditional first app in any language.
+
+## Instructions
+
+1. Create file `hello-world.js` and open it in your favorite editor
+2. Implement an expression (function with `return`) which returns "Hello World"
+3. Export your expression (function with `return`) with `module.exports = ...`
+5. Install testing dependencies with `$ npm install` (`$` is a symbol which tells you that this is a command line command). You need to do it just once for each lab.
+6. Run tests with `$ npm test`.
+
+Tip: Don't overthink it.
+
+### Extra Info
+
+* [Console in Node official documentation](https://nodejs.org/api/console.html)
+
+## Solution
```

Comment:
don't include the solution in the Readme. Student's have access to the solution branch because the curriculum is open-sourced. Just put the solution in `hello-world.js`.

The tests look perfect here!

make sure this readme get's imported to the wip-master branch, right now it has the old version